lib/src/fnc/type.rs

```diff
@@ -94,7 +94,7 @@ pub fn string(_: &Context, mut args: Vec<Value>) -> Result<Value, Error> {
 
 pub fn table(_: &Context, mut args: Vec<Value>) -> Result<Value, Error> {
 	Ok(Value::Table(Table(match args.remove(0) {
-		Value::Thing(t) => t.tb.clone(),
+		Value::Thing(t) => t.tb,
```

Comment:
remove redundant clone I accidentally introduced in a previous PR

Reply:
@finnbear remove this from the PR ... I'm pushing this up separately, just so it fixes clippy warning in the build ...